src/v3/src/components/Widget/index.tsx

```diff
@@ -307,32 +303,29 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
       languageDirection,
     }}
     >
-      {/* Note: we need to keep both themeproviders (MUI/ODS) until ODS exports all MUI components */}
-      <MuiThemeProvider theme={brandedTheme}>
-        <OdysseyCacheProvider nonce={cspNonce}>
-          {/* the style is to allow the widget to inherit the parent's bg color */}
-          <ScopedCssBaseline sx={{ backgroundColor: 'inherit' }}>
-            <OdysseyMuiThemeProvider theme={brandedTheme}>
-              <AuthContainer>
-                <AuthHeader
-                  logo={logo}
-                  logoText={logoText}
-                  brandName={brandName}
-                  authCoinProps={buildAuthCoinProps(idxTransaction)}
-                />
-                <AuthContent>
-                  <IdentifierContainer />
-                  {
-                    uischema.elements.length > 0
-                      ? <Form uischema={uischema as UISchemaLayout} />
-                      : <Spinner />
-                  }
-                </AuthContent>
-              </AuthContainer>
-            </OdysseyMuiThemeProvider>
-          </ScopedCssBaseline>
-        </OdysseyCacheProvider>
-      </MuiThemeProvider>
+      <OdysseyCacheProvider nonce={cspNonce}>
```

Comment:
how come we needed to move the `OdysseyCacheProvider` out?

Reply:
Oh, this isn't moving the cacheprovider out but removing the old legacy theme provider, so it leaves the cacheprovider at the top level now

Reply:
ohh okay got it!